packages/map/src/GMap.ts

```diff
@@ -770,6 +781,9 @@ export class GMap extends HTMLWidget {
                 JSON.stringify(this._userShapes.save()));
         }
     }
+    statusErrorCallback(response) {
+
```

Comment:
By default this method should have the default code:  `console.error("Street View data not found for this location.");`